mulog-kinesis/src/com/brunobonacci/mulog/publishers/aws_utils.clj

```diff
@@ -0,0 +1,15 @@
+(ns com.brunobonacci.mulog.publishers.aws-utils
+  (:require
+    [cognitect.aws.client.api :as aws]))
+
+(def kinesis (aws/client {:api  :kinesis}))
+
+(defn publish!
+  [stream-name records]
+  (aws/invoke kinesis {:op      :PutRecords
+                       :request {:StreamName stream-name
+                                 :Records    records}
+                       }))
+(defn create-records!
```

Comment:
this function doesn't have a side effect so why the bang `!` ???

Reply:
agree, fixed